crates/extensions/c8y_mapper_ext/src/operations/mod.rs

```diff
@@ -374,185 +512,104 @@ fn get_smartrest_response_for_upload_result(
 
 #[cfg(test)]
 mod tests {
-    use crate::tests::skip_init_messages;
-    use crate::tests::spawn_c8y_mapper_actor;
-    use crate::tests::TestHandle;
-    use std::time::Duration;
-    use tedge_actors::test_helpers::MessageReceiverExt;
-    use tedge_actors::Sender;
-    use tedge_mqtt_ext::test_helpers::assert_received_contains_str;
-    use tedge_mqtt_ext::MqttMessage;
-    use tedge_mqtt_ext::Topic;
+    use c8y_auth_proxy::url::Protocol;
+    use c8y_http_proxy::messages::C8YRestRequest;
+    use c8y_http_proxy::messages::C8YRestResult;
+    use tedge_actors::test_helpers::FakeServerBox;
+    use tedge_actors::test_helpers::FakeServerBoxBuilder;
+    use tedge_actors::Builder;
+    use tedge_actors::MessageSink;
+    use tedge_actors::SimpleMessageBox;
+    use tedge_actors::SimpleMessageBoxBuilder;
     use tedge_test_utils::fs::TempTedgeDir;
 
-    const TEST_TIMEOUT_MS: Duration = Duration::from_millis(3000);
+    use crate::tests::test_mapper_config;
+
+    use super::*;
 
     #[tokio::test]
-    async fn mapper_converts_config_metadata_to_supported_op_and_types_for_main_device() {
```

Comment:
This test has been moved to `tests.rs`, while other tests has been moved out of `tests.rs` and dispatched over misc `xxx.rs`.

We should really avoid to do that on a refactoring PR. Reorganize either the code or the tests, not both simultaneously. This makes things harder to understand & check for little benefits.